core/tests/test_utils.py

```diff
@@ -124,6 +124,30 @@ def get_filepath_from_filename(filename, rootdir):
     return filepath
 
 
+def assert_same_list_elements(phrases, stdout):
+    """Checks to see if all of the phrases appear in at least one of the
+    stdout outputs.
+
+    Args:
+        phrases: list(str). A list of phrases we are trying to find in
+        one of the stdout outputs. For example, python linting
```

Comment:
Please indent hanging lines of arg docs by 4 spaces.

Reply:
Done.